### PR TITLE
add priority class for extension pods

### PR DESCRIPTION
**Pull request by danielfoehrKn**
**How to categorize this PR?**

/area high-availability
/kind enhancement
/priority normal
/platform gcp

**What this PR does / why we need it**:
Adds priority class for extension pods to prevent preemption.
Has same priority as the gardenlet class.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:
We do not know which version the Seed is running on

**Release note**:

```improvement operator
Adds priority class for extension pods to prevent preemption.
```

### CODE REVIEW

**Review by rfranzke — APPROVED** [review]
/lgtm

**Review by prashanth26 — APPROVED** [review]
/lgtm

File: charts/gardener-extension-provider-gcp/templates/priorityclass.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+apiVersion: {{ include "priorityclassversion" . }}
+kind: PriorityClass
+metadata:
+  name: gardener-extension-provider-gcp
+value: 1000000000
```
Comment (timebertt) [inline charts/gardener-extension-provider-gcp/templates/priorityclass.yaml:5]: Same question as in https://github.com/gardener/gardener-extension-provider-azure/pull/181#discussion_r510122959

Comment (danielfoehrKn) [reply charts/gardener-extension-provider-gcp/templates/priorityclass.yaml:5]: reolved.

**Review by timebertt — APPROVED** [review]

**Pull request merged**